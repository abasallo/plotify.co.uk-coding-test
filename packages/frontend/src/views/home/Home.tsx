import Companies from '../../components/Companies/Companies'

import { withMainContainer } from '../../templates/withMainContainer'

const Home = () => <Companies />

export default withMainContainer(Home)
