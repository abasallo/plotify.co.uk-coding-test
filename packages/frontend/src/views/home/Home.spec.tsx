import { BrowserRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'

import Home from './Home'
import React from 'react'

describe('Home View', () => {
  test('Renders properly', () => {
    expect(renderer.create(
      <BrowserRouter>
        <Home />
      </BrowserRouter>
    ).toJSON()).toMatchSnapshot()
  })
})
