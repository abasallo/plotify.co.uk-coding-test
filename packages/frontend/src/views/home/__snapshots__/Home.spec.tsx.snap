// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home View Renders properly 1`] = `
<div
  className="MuiContainer-root sc-eDDNvR KMbss MuiContainer-maxWidthLg"
>
  <header
    className="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary sc-iAEyYk hKQuvX MuiPaper-elevation4"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular sc-fsQiph cvNwpl MuiToolbar-gutters"
    >
      <img
        className="sc-qRumB dlUQUu"
        onClick={[Function]}
        src="logo.png"
      />
    </div>
  </header>
  <div
    className="MuiContainer-root sc-jTrPJq ceOdjZ MuiContainer-maxWidthLg"
  >
    <div
      className="MuiContainer-root sc-beqWaB emoJJV MuiContainer-maxWidthLg"
    >
      <div
        className="MuiGrid-root sc-gueYoa gcjcJu MuiGrid-container MuiGrid-spacing-xs-2"
      />
    </div>
  </div>
  <div
    className="MuiBox-root MuiBox-root-1 sc-gLDzan gAcRhg"
  >
    <p
      className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
    >
      Copyright Â© 
      <a
        className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
        href="https://www.linkedin.com/in/abasallo/"
        onBlur={[Function]}
        onFocus={[Function]}
      >
        alvaro@basallo.es
      </a>
       2023
    </p>
  </div>
</div>
`;
